php/PhpJsonApiSamples/web.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <system.webServer>
+        <staticContent>
+            <mimeMap fileExtension=".pem" mimeType="application/x-pem-file" />
+        </staticContent>
+    </system.webServer>

```

Comment:
Fine having web.config file too and I wouldn't remove it. I don't really know the best option here so no harm in leaving it. I just know I wouldn't be developing PHP on a windows/IIS environment.